Minor suggestions for improving the text (#37)
vsoch/gridtest: paper/paper.md

## Conclusion

GridTest aims to be a general tool for data scientists and research software engineers
alike. If there is a need to create a collection of grids, regardless of being used
for testing or another use case, GridTest can solve this problem. By way of saving

> to json, GridTest can represent grids separately from code.

Reviewer: ```suggestion
to yaml, GridTest can represent grids separately from code.
```

> to json, GridTest can generate grids without adding itself as a dependency to your code.

Reviewer: ```suggestion
to json, GridTest can represent grids separately from code.
```
